add message defaults to .ok() and .notOk()

paulcpederson opened the pull request:
Just noticed in a reporter that there are really nice test defaults for methods like `.equals`, but `.ok()` and `.notOk()` just say `(unnamed assert)`.

This adds the following default messages:

- `.ok()` - "should be truthy"
- `.notOk()` - "should be falsey"

Also updates the tests where they are expecting the old default value.

ljharb commented on lib/test.js:
```diff
@@ -312,7 +312,7 @@ Test.prototype.notOk
 = function (value, msg, extra) {
     this._assert(!value, {
-        message : msg,
+        message : defined(msg, 'should be falsey'),
```
nit: `falsy` shouldn't have an `e` imo.

ljharb replied:
also i don't see any tests covering the falsy case

paulcpederson commented:
@ljharb ok, updated to `falsy`. None of the tests threw an error when I changed the notOk error message. Does that mean there are currently no tests for `notOk`?

ljharb commented:
@paulcpederson that's the conclusion i would also come to. would you mind adding some?

fwiw I really like this change - it doesn't change the API, just makes two of the assertions more useful by default.

paulcpederson commented:
cool. I *think* I can figure out how to do that.

paulcpederson commented:
@ljharb ok I added a new test that just tests all of the defaults. Sorry that took me a bit to figure out the test structure. Testing a testing framework is very meta.

ljharb commented:
Awesome, this LGTM as a semver-minor change.

Any other collaborators object? If not, I'll merge this in a day or two.

paulcpederson commented:
@ljharb bump.

Have any other collaborators taken a look?

ljharb commented:
Thanks for the bump - I'll merge this today or tomorrow.

ljharb commented:
@paulcpederson actually would you mind rebasing this on the latest master?

paulcpederson commented:
Not at all.

ljharb commented:
Thanks, ping me when it's updated and I'll merge it!

paulcpederson commented:
@ljharb ok, should be rebased now from `master` thanks for your patience, things got a little crazy today!
